lib/feeder/api.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Feeder.Api do
+
+  @per 100
+  @max_pages 4
+  @ever "1970-01-01"
+
+  def podcasts(), do: get_items("/api/v1/podcasts", %{since: @ever, per: @per})
+  def podcasts(since), do: get_items("/api/v1/podcasts", %{since: format(since), per: @per})
+
+  def episodes(), do: get_items("/api/v1/episodes", %{since: @ever, per: @per})
+  def episodes(pid) when is_integer(pid), do: get_items("/api/v1/podcasts/#{pid}/episodes", %{since: @ever, per: @per})
+  def episodes(since), do: get_items("/api/v1/episodes", %{since: format(since), per: @per})
+  def episodes(pid, since), do: get_items("/api/v1/podcasts/#{pid}/episodes", %{since: format(since), per: @per})
+
+  def get_items(path, params, depth \\ 1) do
```

Comment:
This gets a bit crazy - but what it's doing is following the `next` link to load multiple pages of podcasts/episodes.  But it stops at a max-page depth, and just returns what it's got to the caller.  That way, it doesn't run out of memory trying to load every single page of episodes or something.

So the return value here is one of:

```
{:error, "something"}
{:ok, 100, [array-of-100-items]}
{:partial, 102, [array-of-100-items]}
```